pages/edit-trip/[slug].js

```diff
@@ -82,39 +77,41 @@ export default function EditTrip({ tripsList, setTripsList }) {
             <h2 id="description">Trip Days</h2>
           </ContainerCenterElement>
 
-          {activities.map((activity, index) => (
-            <StyledFieldSet aria-describedby="description" key={index}>
-              <StyledLegend>{`Day ${index + 1}`}</StyledLegend>
-
-              <label htmlFor={`title${index}`}>Title:</label>
-              <input
-                type="text"
-                name={`title${index}`}
-                id={`title${index}`}
-                maxLength={60}
-                value={editedTitles[index] || ""}
-                onChange={(event) => {
-                  const updatedTitles = [...editedTitles];
-                  updatedTitles[index] = event.target.value;
-                  setEditedTitles(updatedTitles);
-                }}
-              />
-
-              <label htmlFor={`activity${index}`}>Activities:</label>
-              <textarea
-                name={`activity${index}`}
-                id={`activity${index}`}
-                rows={4}
-                maxLength={500}
-                value={editedActivities[index] || ""}
-                onChange={(event) => {
-                  const updatedActivities = [...editedActivities];
-                  updatedActivities[index] = event.target.value;
-                  setEditedActivities(updatedActivities);
-                }}
-              ></textarea>
-            </StyledFieldSet>
-          ))}
+          {tripToUpdate &&
+            tripToUpdate.dayDetails &&
```

Comment:
You could also use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here, saves some lines of code ;) 
e.g.
```js
tripToUpdate.dayDetails?.activities?.map(...)
```